### PR TITLE
SONAR-8728 do index create/drop in dedicated Ruby migrations

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DatabaseVersion.java#L128](https://github.com/SonarSource/sonarqube/blob/2a47565f6d438ee2d911d0564e0689b88b5c77c7/sonar-db/src/main/java/org/sonar/db/version/DatabaseVersion.java#L128): Either log or rethrow this exception. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [paging](https://github.com/SonarSource/sonarqube/blob/2a47565f6d438ee2d911d0564e0689b88b5c77c7/server/sonar-server/src/main/java/org/sonar/server/paging): Add a 'package-info.java' file to document the 'paging' package [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [ws](https://github.com/SonarSource/sonarqube/blob/2a47565f6d438ee2d911d0564e0689b88b5c77c7/server/sonar-server/src/main/java/org/sonar/server/updatecenter/ws): Add a 'package-info.java' file to document the 'ws' package [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Comment by julienlancelot** [comment]
Arg, Rails is still there !!!

File: server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1155_add_index_project_measures.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+#
+# SonarQube 5.6.4
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1155_add_index_project_measures.rb]: should be 5.6.6

File: server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1157_drop_index_project_measures.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+#
+# SonarQube 5.6.4
```
Comment (simonbrandhof) [inline server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1157_drop_index_project_measures.rb]: 5.6.6

**Comment by simonbrandhof** [comment]
Successfully tested.

**Pull request merged**